package.json

```diff
@@ -25,7 +25,7 @@
     "ws": "^8.0.0"
   },
   "peerDependencies": {
-    "zwave-js": "^8.0.8"
+    "zwave-js": "8.1.0"
```

Comment:
Did you switch to fixed versioning on purpose? Not saying that it is bad - just caught my eye.

Reply:
nope!